lib/takeaway.rb

```diff
@@ -0,0 +1,36 @@
+class Takeaway
+  attr_reader :menu_class, :order_class
+  def initialize(order_class = Order, menu_class = Menu,
+      send_message_class = SendMessage)
+    @menu_class = menu_class.new
+    @order_class = order_class.new
+    @send_message_class = send_message_class.new
+  end
+
+  def display_menu
+    menu_class.list_menu
+  end
+
+  def ordered_items(item_chosen, quantity)
+    @order_class.select_dishes(item_chosen, quantity)
+  end
+
+  def send_text_message
```

Comment:
This feels like it could probably be a private method.  Unless you expect either a user or another object to make direct use of a method then a good rule of thumb is to always default to private.